platform/binance/token.go

```diff
@@ -7,50 +7,48 @@ import (
 )
 
 func (p *Platform) GetTokenListByAddress(address string) (blockatlas.TokenPage, error) {
-	account, err := p.dexClient.GetAccountMetadata(address)
+	account, err := p.client.GetAccountMetadata(address)
 	if err != nil || len(account.Balances) == 0 {
 		return []blockatlas.Token{}, nil
 	}
-	tokens, err := p.dexClient.GetTokens()
+	tokens, err := p.client.GetTokens()
 	if err != nil {
 		return nil, err
 	}
 	return NormalizeTokens(account.Balances, tokens), nil
 }
 
+// NormalizeTxs converts multiple Binance tokens
+func NormalizeTokens(srcBalance []Balance, tokens *TokenPage) (tokenPage []blockatlas.Token) {
+	for _, srcToken := range srcBalance {
+		token, ok := NormalizeToken(&srcToken, tokens)
+		if !ok {
+			continue
+		}
+		tokenPage = append(tokenPage, token)
+	}
+	return
+}
+
 // NormalizeToken converts a Binance token into the generic model
 func NormalizeToken(srcToken *Balance, tokens *TokenPage) (t blockatlas.Token, ok bool) {
 	if srcToken.isAllZeroBalance() {
 		return t, false
 	}
 
-	tk := tokens.findToken(srcToken.Symbol)
+	tk := tokens.findToken(srcToken.Symbol) // TODO create and use cache instead query tokens list every time
```

Comment:
You can use GetWithCache to make HTTP GET request 
Example:
https://github.com/trustwallet/watchmarket/blob/master/market/clients/cmc/client.go#L49

Reply:
@kolya182 remove TODO if you added cache